.editorconfig

```diff
@@ -149,7 +149,7 @@ ij_java_method_parameters_right_paren_on_new_line = false
 ij_java_method_parameters_wrap = off
 ij_java_modifier_list_wrap = false
 ij_java_multi_catch_types_wrap = normal
-ij_java_names_count_to_use_import_on_demand = 3
+ij_java_names_count_to_use_import_on_demand = 30
```

Comment:
the above changes will end up with solving the annoying `import static wildcard` issue which troubled me all the time.
Spotless decides to ignore such common issue due to backward compatibility: https://github.com/diffplug/spotless/issues/649, unfortunately